internal/service/verify/data_source_verify_voice_phrase_contents.go

```diff
@@ -0,0 +1,184 @@
+package verify
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"regexp"
+
+	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/diag"
+	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/patrickcping/pingone-go-sdk-v2/pingone/model"
+	"github.com/patrickcping/pingone-go-sdk-v2/verify"
+	"github.com/pingidentity/terraform-provider-pingone/internal/framework"
+	"github.com/pingidentity/terraform-provider-pingone/internal/sdk"
+	validation "github.com/pingidentity/terraform-provider-pingone/internal/verify"
+)
+
+// Types
+type VoicePhraseContentsDataSource struct {
+	client *verify.APIClient
+	region model.RegionMapping
+}
+
+type voicePhraseContentsDataSourceModel struct {
+	Id            types.String `tfsdk:"id"`
+	EnvironmentId types.String `tfsdk:"environment_id"`
+	VoicePhraseId types.String `tfsdk:"voice_phrase_id"`
+	Ids           types.List   `tfsdk:"ids"`
+}
+
+// Framework interfaces
+var (
+	_ datasource.DataSource = &VoicePhraseContentsDataSource{}
+)
+
+// New Object
+func NewVoicePhraseContentsDataSource() datasource.DataSource {
+	return &VoicePhraseContentsDataSource{}
+}
+
+// Metadata
+func (r *VoicePhraseContentsDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_verify_voice_phrase_contents"
+}
+
+func (r *VoicePhraseContentsDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+
+	// schema descriptions and validation settings
+
+	// P1 Platform does not set a traditional UUID as the default phrase ID value
+	const defaultVoicePhraseId = "exceptional_experiences"
```

Comment:
Removed - the data source will only allow a `voice_phrase_id` using the standard UUID format.

Configuring the default phrases in the provider is not allowed because the phrase ids are non-standard, simple examples, and may change in the future.  An implementer is also prevented from using the data source to query the non-standard, default example `voice_phrase` objects.

Disallowing the implementer from using the pre-loaded example phrases (which are product-focused, and may change in the future) prevents reliance on non-standard UUIDs and configuration objects that may change in the future outside of the implementer's control.